### ISSUE TITLE
Node number 241 (TfLiteFlexDelegate) failed to invoke.

### ISSUE THREAD

**Issue by MannyKai** [tensorflow/tensorflow#42185]
**System information**
- OS Platform  : TFlite C++ API on Android Platform
- TensorFlow version : tf-nightly 2.4.0

**I have build tflite C library with Flex Op module.

But when I test it with a Tflite model which contain flex ops, I have got a error while invoke：**

 op_kernel.cc:1772 OP_REQUIRES failed at tensor_array_ops.cc:1035 : Not found: Container __per_step_0 does not exist. (Could not find resource: __per_step_0/_tensor_arraysTensorArrayV3_0)
Container __per_step_0 does not exist. (Could not find resource: __per_step_0/_tensor_arraysTensorArrayV3_0)
    	 (while executing 'TensorArrayScatterV3' via Eager)
Node number 241 (TfLiteFlexDelegate) failed to invoke.

Has anyone encountered such a problem?

**Comment by amahendrakar** [comment]
@MannyKai,
In order to expedite the trouble-shooting process, could you please provide the complete code to reproduce the issue reported here. Thanks!

**Comment by MannyKai** [comment]
> @MannyKai,
> In order to expedite the trouble-shooting process, could you please provide the complete code to reproduce the issue reported here. Thanks!

Yes, I have re submitted the question and described it in more detail，[https://github.com/tensorflow/tensorflow/issues/42214](url)

**Comment by amahendrakar** [comment]
@MannyKai,
Can we please close this issue since it is already being tracked there? Thanks!
